DependencyInjection/SonataPageExtension.php

```diff
@@ -499,73 +495,4 @@ public function configurePageServices(ContainerBuilder $container, array $config
         $definition = $container->getDefinition('sonata.page.page_service_manager');
         $definition->addMethodCall('setDefault', [new Reference($config['default_page_service'])]);
     }
-
-    /**
-     * Add class to compile.
-     */
-    public function configureClassesToCompile()
```

Comment:
Technical, it's a BC break. Not sure if we could accept this, 'cause this method should never been called in this context

Reply:
It's fine.